apps/meteor/app/apps/server/converters/threads.ts

```diff
@@ -101,6 +103,7 @@ export class AppThreadsConverter {
 			groupable: 'groupable',
 			token: 'token',
 			blocks: 'blocks',
+			// #ToDo: #AppsEngineTypes - bug? - This "room" object won't be properly mapped to the app object
```

Comment:
Possible bug on apps-engine: Room object being passed to the map won't be copied to the resulting object